web-app/django/VIM/apps/main/views.py

```diff
@@ -7,3 +9,45 @@ def home(request):
 
 def about(request):
     return render(request, "main/about.html", {"active_tab": "about"})
+
+
+def user_login(request):
```

Comment:
Can we use the default authentication views: 

https://docs.djangoproject.com/en/4.2/topics/auth/default/#module-django.contrib.auth.views